<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createNewProject</class>
 <widget class="QWidget" name="newFKProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>311</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New Project</string>
  </property>
  <widget class="QPushButton" name="Back_button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>240</y>
     <width>85</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel_button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>270</y>
     <width>85</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Create_button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>210</y>
     <width>85</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>481</width>
     <height>261</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="ProjectTab">
    <attribute name="title">
     <string>Project</string>
    </attribute>
    <widget class="QLineEdit" name="projectName">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>newProject</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>66</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Create in:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="project_dir">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>./newProject</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Browse_new">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>60</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="DatabaseTab">
    <attribute name="title">
     <string>Database</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Database directory:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="database_dir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>orl_faces</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Browse_button">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>40</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
